website/docs/language/functions/nonsensitive.mdx

```diff
@@ -73,8 +73,8 @@ due to an inappropriate call to `nonsensitive` in your module, that's a bug in
 your module and not a bug in OpenTofu itself.
 **Use this function sparingly and only with due care.**
 
-`nonsensitive` will return an error if you pass a value that isn't marked
-as sensitive, because such a call would be redundant and potentially confusing
+`nonsensitive` will no longer return an error if you pass a value that isn't marked
```

Comment:
I think we should not reference the past in docs this way ("no longer") - for a person who hasn't used this before, it doesn't make much sense.

I'd propos just saying "will not return".

Reply:
Agreed, this is the only thing I see to address before this is merged.